### ISSUE TITLE
Searching doesn't show any results on large projects without relaunching Code

### ISSUE THREAD

**Issue by ChristopherHammond13** [Microsoft/vscode#66490]
Version: 1.31.0-insider (system setup)
Commit: 2ab82c12ed809fd1df921d016eed344a18a9ee8f
Date: 2019-01-14T06:18:47.017Z
Electron: 3.0.13
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

Steps to Reproduce:

1. Load a large project (I am using Nginx and OpenSSL together)
2. Do a handful of searches
3. Make some file edits
4. Come back later and do a search
5. Results sometimes (but not always) start to show up
6. Once the search finishes, the stop button stays available and any results that have been shown vanish from the screen

Additionally, inconsistent state in the stop/clear/collapse buttons causes crashes when the buttons are clicked in a certain order.

Both bugs are visible in this gif.
![search bug](https://user-images.githubusercontent.com/295396/51122817-9b919c80-1812-11e9-9780-ed4011ed4527.gif)

Does this issue occur when all extensions are disabled?: Yes

Thanks so much for your help! I hope this bug report helps. It's my first for VS Code so apologies if I have missed off any important information.

**Comment by roblourens** [comment]
This is similar to https://github.com/Microsoft/vscode/issues/66165#issuecomment-453995423, I can't figure out what exactly puts us into this broken state.

> To fix the bug, I need to close every tab and completely close VS Code, then reopen the directory. Then search temporarily works again. Reopening, or pressing the button to reload without extensions enabled, doesn't work.

Except for this which I don't totally understand - when I have seen this bug, it has been fixed by a simple Reload Window

**Comment by joaomoreno** [comment]
Yep pretty sure it's the same as https://github.com/Microsoft/vscode/issues/66573
